### PR TITLE
Set rollout strategy to Recreate for prometheus-operator

### PR DESCRIPTION
**Pull request by marieroque**
towards https://github.com/giantswarm/giantswarm/issues/29665

### CODE REVIEW

File: helm/kube-prometheus-stack/values.yaml [hunk 1]
```diff
@@ -362,6 +362,9 @@ kube-prometheus-stack:
   prometheusOperator:
     enabled: true
     admissionWebhooks:
+      deployment:
```
Comment (QuentinBisson) [inline helm/kube-prometheus-stack/values.yaml:365]: That's for admission webhook though right?

Comment (QuentinBisson) [reply helm/kube-prometheus-stack/values.yaml:365]: Not the actual operator?

**Comment by marieroque** [comment]
- [ ] tested on `gaia`
- [ ] tested on `golem`

**Review by QuentinBisson — APPROVED** [review]

**Pull request merged**